wakuv2/waku.go

```diff
@@ -1275,16 +1279,27 @@ func (w *Waku) Stop() error {
 
 	w.node.Stop()
 
+	time.Sleep(2 * time.Second)
+	w.logger.Info("<<< push to telemetry client")
+
+	if w.statusTelemetryClient != nil {
+		w.statusTelemetryClient.PushPeerCount(w.PeerCount())
```

Comment:
This is not needed, right?

Reply:
Yeah this is leftover from the way we had to trigger the freeze

Reply:
oops

Reply:
452fb5e078dec81bdaf2460eb292b92f3bde956c